cabal.project.stage2

```diff
@@ -128,6 +128,22 @@ if os(darwin)
     flags: +tables-next-to-code +leading-underscore
 	ld-options: -undefined warning
 
+if os(freebsd)
+  package rts
+    ghc-options: "-optc-DProjectVersion=\"914\""
+    ghc-options: "-optc-DRtsWay=\"v\""
+    ghc-options: "-optc-DHostPlatform=\"x86_64-portbld-freebsd\""
+    ghc-options: "-optc-DHostArch=\"x86_64\""
+    ghc-options: "-optc-DHostOS=\"freebsd\""
+    ghc-options: "-optc-DHostVendor=\"unknown\""
+    ghc-options: "-optc-DBuildPlatform=\"FIXME\""
+    ghc-options: "-optc-DBuildArch=\"FIXME\""
+    ghc-options: "-optc-DBuildOS=\"FIXME\""
+    ghc-options: "-optc-DBuildVendor=\"FIXME\""
+    ghc-options: "-optc-DGhcUnregisterised=\"FIXME\""
+    ghc-options: "-optc-DTablesNextToCode=\"FIXME\""
+    ghc-options: "-optc-DFS_NAMESPACE=rts"
+    flags: +tables-next-to-code
```

Comment:
These should be extracted into a separate PR: "FreeBSD support".

Reply:
Don't see why.